markdown/GameGuides/dark_souls_3.md

```diff
@@ -14,12 +14,23 @@ Graphics API | DirectX 11
  
 ## Tools
 
-* [CineTools by Hattiwatti](https://mega.nz/#!8Jg2TSjA!agNohvMiOD8xhv_IHiI21Af9J39FA8bAOIUOsQYzb-M)  
+* [CineTools by Hattiwatti Password: CinematicTools](https://www.mediafire.com/file/gsc2ri2ehhqmma5/DSIII_20160420-aa09dd.rar/file)
 **Features**: Camera control, FoV
-* [Challenge Mods](https://www.nexusmods.com/darksouls3/mods/136)  

```

Comment:
This mod doesn't work anymore?

Reply:
not with the current game version 1.15.2